File: 3188-find-champion-i/find-champion-i.py
class Solution:
    def findChampion(self, grid: List[List[int]]) -> int:
        n = len(grid)
        for i in range(n):
            is_champion = True
            for j in range(n):
                if grid[j][i] == 1:
                    is_champion = False
                    break
            if is_champion:
                return i
        return -1

File: 3188-find-champion-i/find-champion-i.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
var findChampion = function(grid) {
    let champion = 0;
    for (let i = 1; i < grid.length; i++) {
        if (grid[i][champion] === 1) {
            champion = i;
        }
    }
    return champion;
};